app/components/task_list_items/assessment/meeting_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module TaskListItems
+  module Assessment
+    class MeetingComponent < TaskListItems::BaseComponent
+      def initialize(planning_application:)
+        @planning_application = planning_application
+      end
+
+      private
+
+      attr_reader :planning_application
+
+      delegate :meeting, to: :planning_application
+
+      def link_text
+        "Meeting"
+      end
+
+      def link_path
+        if @planning_application.meetings.any?
+          planning_application_assessment_meetings_path(@planning_application)
+        else
+          new_planning_application_assessment_meeting_path(@planning_application)
+        end
+      end
+
+      def status_tag_component
+        if @planning_application.meetings.any?
+          StatusTags::BaseComponent.new(
+            status: @planning_application.meetings.last.status
+          )
+        else
+          StatusTags::BaseComponent.new(
+            status: "not_started"
+          )
+        end
```

Comment:
This could also be written with a ternary, which would be a little shorter:

```suggestion
        StatusTags::BaseComponent.new(
          status: @planning_application.meetings.any? ? @planning_application.meetings.last.status : "not_started"
        )
```